src/components/DonutChart.js

```diff
@@ -9,6 +10,8 @@ const DonutChart = React.createClass({
     activeIndex: React.PropTypes.number,
     activeOffset: React.PropTypes.number,
     animateOnHover: React.PropTypes.bool,
+    animationDuration: React.PropTypes.number,
```

Comment:
I think we should standardize the naming of these for all our animation components. I was just using `duration` in the drawer but I like the animation prefix better. I'll incorporate that into my next Drawer pr.